Extension/src/common.ts

```diff
@@ -1276,3 +1276,11 @@ export function isVsCodeInsiders(): boolean {
         extensionPath.includes(".vscode-exploration") ||
         extensionPath.includes(".vscode-server-exploration");
 }
+
+export function isAlpine(): boolean {
+    const output: string = child_process.spawnSync('ldd', ['--version']).stderr.toString();
```

Comment:
We should already have the information we need in the LinuxDistribution class (`distroName` field).  According to [this](https://linuxhandbook.com/check-linux-version/#:~:text=The%20os-release%20file%20in%20the%20%2Fetc%20directory%20keeps,for%20Alpine%20Linux%20server%20running%20on%20Linode%20infrastructure.) the ID will be 'alpine'.

Reply:
I think it would be safer to use that than this approach.